lib/tools/apk-utils.js

```diff
@@ -117,8 +117,15 @@ apkUtilsMethods.waitForActivityOrNot = async function (pkg, activity, not,
   log.debug(`Waiting for pkg: '${pkg}' and activity: '${activity}'` +
             `${not ? ' not' : ''} to be focused`);
   let endAt = Date.now() + waitMs;
-  let possibleActivityNames = getPossibleActivityNames(pkg, activity);
+  
+  let possibleActivityNames = [];
+  let allActivities = activity.split(",");
+  allActivities.forEach(function (oneActivity) {
+    possibleActivityNames.push(getPossibleActivityNames(pkg, oneActivity.trim()));
+  });
+  possibleActivityNames = [].concat.apply([], possibleActivityNames);
```

Comment:
The return from getPossibleActivityNames() is an array. So the elements in possibleActivityNames are arrays and they will fail the _.findIndex() further down unless possibleActivityNames is flattened which is what this line does.